### PR TITLE
[fixed My Wish List Product not showing properly between >768px and <…

### PR DESCRIPTION
**Pull request by hitesh-wagento**
### Description (*)
Fixed My Wish List Product not showing properly between >768px and <1023px

### Fixed Issues (if relevant)

1. magento/magento2#26543: My Wish List Product not showing properly between >768px and <1023px

### Manual testing scenarios (*)

1. Login in frontend
2. Add 4-5 products in wishlist
3. Go to Wishlist page

### Expected result

Products should show 3 items
expected

### Actual result
Products show 2 products and space right side

actual

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Review by ptylek — CHANGES_REQUESTED** [review]
Hello, @hitesh-wagento, thank you for your contribution. 
I don't understand why did you choose those values for margin and width. Also later on, they are overwritten by width: 100% and margin: 0 anyway, aren't they?

**Comment by hitesh-wagento** [comment]
> Hello, @hitesh-wagento, thank you for your contribution.
> I don't understand why did you choose those values for margin and width. Also later on, they are overwritten by width: 100% and margin: 0 anyway, aren't they?

Hi @ptylek 

I have applied margin as per desktop and >640px; and after <640px you can see 100% width not required margin.

Thanks

**Comment by ptylek** [comment]
Okay, I understand @hitesh-wagento. But why these values such as margin-left: **2%**; padding: **5px**; width: calc(~'(**100% - 4%**) / 3')? I couldn't find any other grid using these values. Aren't there any variables?

**Comment by hitesh-wagento** [comment]
> Okay, I understand. But why these values such as margin-left: **2%**; padding: **5px**; width: calc(~'(**100% - 4%**) / 3')? I couldn't find any other grid using these values. Aren't there any variables?

Hi @ptylek 

You can find this variable in category pages on a smaller size in the below CSS.
`.media-width(@extremum, @break) when (@extremum = 'min') and (@break = @screen__m) {`

`/Magento_Catalog/web/css/source/module/_listings.less`

Thanks

File: app/design/frontend/Magento/blank/Magento_Wishlist/web/css/source/_module.less [hunk 1]
```diff
@@ -53,6 +53,17 @@
         }
     }
 
+    .products-grid.wishlist {
+        .product-item {
+            margin-left: 2%;
+            padding: 5px;
```
Comment (ptylek) [inline app/design/frontend/Magento/blank/Magento_Wishlist/web/css/source/_module.less]: Remove padding property

File: app/design/frontend/Magento/blank/Magento_Wishlist/web/css/source/_module.less [hunk 2]
```diff
@@ -53,6 +53,17 @@
         }
     }
 
+    .products-grid.wishlist {
+        .product-item {
```
Comment (ptylek) [inline app/design/frontend/Magento/blank/Magento_Wishlist/web/css/source/_module.less]: Can these styles be moved to line 128 (.products-grid.wishlist { .product { &-item)?

File: app/design/frontend/Magento/blank/Magento_Wishlist/web/css/source/_module.less [hunk 3]
```diff
@@ -53,6 +53,17 @@
...
+            margin-left: 2%;
+            padding: 5px;
+            width: calc(~'(100% - 4%) / 3');
+            &:nth-child(3n + 1) {
```
Comment (ptylek) [inline app/design/frontend/Magento/blank/Magento_Wishlist/web/css/source/_module.less]: Add empty line before `&:`

File: app/design/frontend/Magento/luma/Magento_Wishlist/web/css/source/_module.less [hunk 4]
```diff
@@ -45,6 +45,17 @@
         }
     }
 
+    .products-grid.wishlist {
+        .product-item {
```
Comment (ptylek) [inline app/design/frontend/Magento/luma/Magento_Wishlist/web/css/source/_module.less]: Can these styles be moved to line 137 (.products-grid.wishlist { .product { &-item)?

File: app/design/frontend/Magento/luma/Magento_Wishlist/web/css/source/_module.less [hunk 5]
```diff
@@ -45,6 +45,17 @@
...
+            margin-left: 2%;
+            padding: 5px;
```
Comment (ptylek) [inline app/design/frontend/Magento/luma/Magento_Wishlist/web/css/source/_module.less]: Remove padding property

File: app/design/frontend/Magento/luma/Magento_Wishlist/web/css/source/_module.less [hunk 6]
```diff
@@ -45,6 +45,17 @@
...
+            width: calc(~'(100% - 4%) / 3');
+            &:nth-child(3n + 1) {
```
Comment (ptylek) [inline app/design/frontend/Magento/luma/Magento_Wishlist/web/css/source/_module.less]: Add empty line before `&:`

File: app/design/frontend/Magento/luma/Magento_Wishlist/web/css/source/_module.less [hunk 7]
```diff
@@ -216,7 +227,8 @@
     .products-grid.wishlist {
         .product-item {
             border-bottom: 1px solid @secondary__color;
-
+            margin: 0;
+            width: 100%;
```
Comment (ptylek) [inline app/design/frontend/Magento/luma/Magento_Wishlist/web/css/source/_module.less:227]: Add empty line after before `&:first-child`

File: app/design/frontend/Magento/blank/Magento_Wishlist/web/css/source/_module.less [hunk 8]
```diff
@@ -185,6 +196,8 @@
     .products-grid.wishlist {
         .product-item {
             border-bottom: 1px solid @secondary__color;
+            margin: 0;
+            width: 100%;
```
Comment (ptylek) [inline app/design/frontend/Magento/blank/Magento_Wishlist/web/css/source/_module.less:196]: Add empty line after before &:first-child

**Review by ptylek — CHANGES_REQUESTED** [review]

**Comment by ptylek** [comment]
@hitesh-wagento, thanks for clarification, I added some comments

File: app/design/frontend/Magento/blank/Magento_Wishlist/web/css/source/_module.less (hunk 1 above)
Comment (hitesh-wagento) [reply app/design/frontend/Magento/blank/Magento_Wishlist/web/css/source/_module.less]: I have applied changes as per your suggestion

File: app/design/frontend/Magento/blank/Magento_Wishlist/web/css/source/_module.less (hunk 2 above)
Comment (hitesh-wagento) [reply app/design/frontend/Magento/blank/Magento_Wishlist/web/css/source/_module.less]: I have applied changes as per your suggestion

File: app/design/frontend/Magento/blank/Magento_Wishlist/web/css/source/_module.less (hunk 3 above)
Comment (hitesh-wagento) [reply app/design/frontend/Magento/blank/Magento_Wishlist/web/css/source/_module.less]: I have applied changes as per your suggestion

File: app/design/frontend/Magento/luma/Magento_Wishlist/web/css/source/_module.less (hunk 4 above)
Comment (hitesh-wagento) [reply app/design/frontend/Magento/luma/Magento_Wishlist/web/css/source/_module.less]: I have applied changes as per your suggestion

File: app/design/frontend/Magento/luma/Magento_Wishlist/web/css/source/_module.less (hunk 5 above)
Comment (hitesh-wagento) [reply app/design/frontend/Magento/luma/Magento_Wishlist/web/css/source/_module.less]: I have applied changes as per your suggestion

File: app/design/frontend/Magento/luma/Magento_Wishlist/web/css/source/_module.less (hunk 6 above)
Comment (hitesh-wagento) [reply app/design/frontend/Magento/luma/Magento_Wishlist/web/css/source/_module.less]: I have applied changes as per your suggestion

File: app/design/frontend/Magento/luma/Magento_Wishlist/web/css/source/_module.less (hunk 7 above)
Comment (hitesh-wagento) [reply app/design/frontend/Magento/luma/Magento_Wishlist/web/css/source/_module.less:227]: I have applied changes as per your suggestion

File: app/design/frontend/Magento/blank/Magento_Wishlist/web/css/source/_module.less (hunk 8 above)
Comment (hitesh-wagento) [reply app/design/frontend/Magento/blank/Magento_Wishlist/web/css/source/_module.less:196]: I have applied changes as per your suggestion

**Review by ptylek — APPROVED** [review]
Great, @hitesh-wagento! Thank you for your contribution.

**Comment by magento-engcom-team** [comment]
Hi @ptylek, thank you for the review.
[ENGCOM-6723](https://jira.corp.magento.com/browse/ENGCOM-6723) has been created to process this Pull Request

**Comment by engcom-Hotel** [comment]
:heavy_check_mark: QA Passed

**Comment by hitesh-wagento** [comment]
#MM20IN

**Comment by hitesh-wagento** [comment]
#mm20in

**Pull request merged**